src/views/components/shared/Input/__tests__/component.spec.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import Input from '../component'
+
+describe('Input component matches snapshot', () => {
```

Comment:
`matches snapshot` should go to the `it` block

Reply:
fixed